app/models/gsa18f/procurement.rb

```diff
@@ -10,6 +10,13 @@ class Procurement < ActiveRecord::Base
     URGENCY = DATA['URGENCY']
     OFFICES = DATA['OFFICES']
     RECURRENCE = DATA['RECURRENCE']
+    PURCHASE_TYPES = {
+      "Software" => 0,
+      "Training/Event" => 1,
+      "Office Supply/Miscellaneous" => 2
+    }
```

Comment:
ok, added tests for documentation here: 6d2481ae2ce0e1667a485a4d9cc7dd4961d17467 

I think this is a good way to ensure people don't change this order because changing would also require a change in specs -- what do you think?